packages/nimble-components/src/table/types.ts

```diff
@@ -0,0 +1,14 @@
+// TableDataValue describes the supported value types for values
+// within the data given to a table.
```

Comment:
Can you format this as doc comments with the double asterick (then it shows up in the editor)

```ts
/ **
  stuff
  */
```